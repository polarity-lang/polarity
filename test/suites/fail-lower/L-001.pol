def Unit.bar() : Unit {
    MkUnit => Foo
}
 