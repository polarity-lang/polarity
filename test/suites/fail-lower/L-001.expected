L-001

  × Undefined identifier MkUnit
   ╭─[L-001.pol:2:5]
 1 │ def Unit.bar() : Unit {
 2 │     MkUnit => Foo
   ·     ──────
 3 │ }
   ╰────
